### ISSUE TITLE
About the title

### ISSUE THREAD

**Issue by LeeBC2298** [MuggleWang/CosFace_pytorch#15]
I think this project is about 'SphereFace', not 'CosFace'.
CosFace use LMCL loss function, not A-softmax loss.

**Comment by fuenwang** [comment]
@LeeBC2298 
I think it's indeed cosface. The output of the classifier (https://github.com/MuggleWang/CosFace_pytorch/blob/master/layer.py#L43) is the cosine similarity with margin. And directly optimizing the cosine similarity with cross entropy loss is equal to LMCL (Eq. 4 in the paper).
